lms/djangoapps/certificates/management/commands/regenerate_user.py

```diff
@@ -23,42 +22,42 @@ class Command(BaseCommand):
 
     help = """Put a request on the queue to recreate the certificate for a particular user in a particular course."""
 
-    option_list = BaseCommand.option_list + (
-        make_option('-n', '--noop',
-                    action='store_true',
-                    dest='noop',
-                    default=False,
-                    help="Don't grade or add certificate requests to the queue"),
-        make_option('--insecure',
-                    action='store_true',
-                    dest='insecure',
-                    default=False,
-                    help="Don't use https for the callback url to the LMS, useful in http test environments"),
-        make_option('-c', '--course',
-                    metavar='COURSE_ID',
-                    dest='course',
-                    default=False,
-                    help='The course id (e.g., mit/6-002x/circuits-and-electronics) for which the student named in'
-                         '<username> should be graded'),
-        make_option('-u', '--user',
-                    metavar='USERNAME',
-                    dest='username',
-                    default=False,
-                    help='The username or email address for whom grading and certification should be requested'),
-        make_option('-G', '--grade',
-                    metavar='GRADE',
-                    dest='grade_value',
-                    default=None,
-                    help='The grade string, such as "Distinction", which should be passed to the certificate agent'),
-        make_option('-T', '--template',
-                    metavar='TEMPLATE',
-                    dest='template_file',
-                    default=None,
-                    help='The template file used to render this certificate, like "QMSE01-distinction.pdf"'),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument('-n', '--noop',
+                            action='store_true',
+                            dest='noop',
+                            default=False,
```

Comment:
Nit: Elsewhere, these `default=False` parms are removed due to it being the default?

Reply:
Dang, thought I got all of those. I'll fix it.